tests/test_clamp_ghosts.py

```
@@ -0,0 +1,330 @@
+"""Tests for `aelfrice.clamp_ghosts` + the `aelf clamp-ghosts` CLI.
+
+Verifies:
+- The structural ghost-α predicate (no feedback_history, no
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `α` (GREEK SMALL LETTER ALPHA). Did you mean `a` (LATIN SMALL LETTER A)?

(RUF002)